app/sagas/login.js

```diff
@@ -127,8 +127,8 @@ const handleLoginSuccess = function* handleLoginSuccess({ user }) {
 			}
 		});
 
-		yield RNUserDefaults.set(`${ RocketChat.TOKEN_KEY }-${ server }`, user.id);
-		yield RNUserDefaults.set(`${ RocketChat.TOKEN_KEY }-${ user.id }`, user.token);
+		yield MMKV.setStringAsync(`${ RocketChat.TOKEN_KEY }-${ server }`, user.id);
```

Comment:
Same as `currentServer`.
It could be a function instead of having to know the pattern.
But both could be on a separate PR.
I'm not sure.

Reply:
I'll do this on a separate PR, soon, create functions on UserPreferences specific to get/set user credentials based on server :)